resources/services/iam/groups_access_advisor.go

```diff
@@ -0,0 +1,188 @@
+package iam
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource groups_access_advisor --config gen.hcl --output .
+func GroupsAccessAdvisors() *schema.Table {
+	return &schema.Table{
+		Name:          "aws_iam_groups_access_advisor",
+		Resolver:      fetchIamGroupsAccessAdvisors,
+		Multiplex:     client.AccountMultiplex,
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountFilter,
+		IgnoreInTests: true,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "service_namespace"}},
+		Columns: []schema.Column{
```

Comment:
Will the table structure differ for IAM Roles/Policies/Users or could we make the table generic and just have specific resolvers for each type? I think that would make it easier for querying/using the data

Reply:
You are right. The structure is the same, but before listing access advisor resources we need to list users, groups, etc. So root fetch should be different. Some of the resolvers can be generic. I have added a commit that does it. Thanks for a hint.